admin/data/sql/migrations/20190816091654.do.create-pupilgroup-archive.sql

```diff
@@ -0,0 +1,9 @@
+
+create table mtc_admin.z_pupilGroup_archive
```

Comment:
@GuyHarwood why not `SELECT INTO` z_pupilGroup_archive from pupilGroup? That would save 1 migration

Reply:
that is used in the next migration to do the data copy.  i like to keep simple units of work per migration.  just my preference 🤷‍♂

Reply:
great idea 🏆